app/src/main/java/me/ccrama/redditslide/Notifications/CheckForMail.java

```diff
@@ -200,26 +201,26 @@ public void onPostExecute(List<Message> messages) {
                                 2 + (int) m.getCreated().getTime(), c,
                                 new String[]{m.getFullName()});
 
-                        Notification notification =
-                                new NotificationCompat.Builder(c).setContentIntent(openPi)
+                        NotificationCompat.Builder builder = new NotificationCompat.Builder(c,
+                                Reddit.CHANNEL_MAIL).setContentIntent(openPi)
                                         .setSmallIcon(R.drawable.notif)
                                         .setTicker(res.getQuantityString(
                                                 R.plurals.mail_notification_title, 1, 1))
                                         .setWhen(System.currentTimeMillis())
                                         .setAutoCancel(true)
                                         .setContentTitle(contentTitle)
-                                        .setChannelId(
```

Comment:
I dropped all of these since the builder method takes in the channel you're building on now. No need to chain with this anymore.